### DISCUSSION TITLE
希望带 Alpha 通道的H265视频 播放时透明像素显示为棋盘 而不是显示黑色

### DISCUSSION THREAD

**Discussion by x1000433** [iina/iina#4632] [General]
希望带 Alpha 通道的H265视频 播放时透明像素显示为棋盘 而不是显示黑色
有不少透明背景的视频都能正常显示，唯独H265的显示为黑色。

**Comment by low-batt** [comment]
母语为英语的用户使用在线翻译进行回复...


在音频/视频支持方面，IINA 使用了 [mpv](https://github.com/mpv-player/mpv) 项目中的一个库。如果 mpv 播放器表现出相同的行为，则必须向该项目报告问题。


我很难找到具有 alpha 通道的视频样本。我在 Apple 示例代码项目 [Using HEVC Video with Alpha](https://developer.apple.com/documentation/avfoundation/media_playback/using_hevc_video_with_alpha) 中找到了一些。


播放 [Apple ProRes](https://en.wikipedia.org/wiki/Apple_ProRes) 样本：

```text
mpv --no-config ~/Movies/puppets_with_alpha_prores.mov 
```


显示 mpv [alpha](https://mpv.io/manual/stable/#options-alpha) 选项中描述的灰/白色色块：
> **--alpha=<blend-tiles|blend|yes|no>**
>
> 如果输入有 alpha 分量，则决定如何处理。
>
 >**blend-tiles**
 > 在 16x16 灰/白瓷砖背景上混合帧（默认）。

![alpha-channel](https://github.com/iina/iina/assets/86170219/d5c09897-83f7-4269-b5e2-49627973a6bf)


播放 HEVC 样本：

文本
```text
mpv --no-config ~/Movies/puppets_with_alpha_hevc.mov
```

显示黑色背景。

因此，这种 IINA 行为来自 mpv，需要向该项目报告。
